e2e/Makefile

```diff
@@ -24,13 +24,13 @@ export TEST_KUBERNETES_VERSION
 KUSTOMIZE_DIR=e2ebase
 
 ifeq ($(TEST_KUBERNETES_VERSION),1.18)
-KUBERNETES_VERSION=1.18.9
+KUBERNETES_VERSION=1.18.8
```

Comment:
Why didn't you use kubernetes 1.18.9?

Reply:
there's no such tag: https://hub.docker.com/r/kindest/node/tags